### DISCUSSION TITLE
Buying a Cup of Coffee

### DISCUSSION THREAD

**Discussion by shawnhaywood** [ufodone/envisalink_new#120] [General]
This is a great custom integration.  As many have recently experienced, the envisalink HA integration has made some changes that aren't desirable to me (I'm sure they will be fixed).  Anyway, someone pointed me in the direction of this custom integration.  I appreciate all the work done.  Is there a link to "buy a cup of coffee"?

**Comment by ufodone** [comment]
Hi, thanks for the note.   I don't have a "buy a cup of coffee" link.  All I ask for is a bit of patience as it sometimes takes me a bit to get around to sorting some issues out.  Cheers!
